src/apollo/Apollo.tsx

```diff
@@ -27,12 +26,12 @@ export const client = new ApolloClient({
     cache,
 });
 
-function ann(){
+function ann() {
   return authMiddleware.concat(createIsomorphLink())
 }
 
 function createIsomorphLink() {
     return new HttpLink({
-      uri: 'http://localhost:4000',
+      uri: 'http://192.168.1.118:4000',
```

Comment:
better not to use local network ip here, localhost or 127.0.0.1 should be fine

Reply:
agreed, my bad, I was playing with it on different devices